WooCommerce/Classes/Extensions/DateFormatter+Helpers.swift

```diff
@@ -8,43 +8,74 @@ extension DateFormatter {
     ///
     struct Charts {
 
+        // MARK: - Chark axis formatters
+
         /// Date formatter used for creating the date displayed on a chart axis for **day** granularity.
         ///
-        public static let chartsDayFormatter: DateFormatter = {
+        public static let chartAxisDayFormatter: DateFormatter = {
             let formatter = DateFormatter()
-            formatter.locale = Locale(identifier: "en_US_POSIX")
-            formatter.timeZone = TimeZone(identifier: "GMT")
-            formatter.dateFormat = "MMM d"
+            formatter.setLocalizedDateFormatFromTemplate("MMM d")
             return formatter
         }()
 
         /// Date formatter used for creating the date displayed on a chart axis for **week** granularity.
         ///
-        public static let chartsWeekFormatter: DateFormatter = {
+        public static let chartAxisWeekFormatter: DateFormatter = {
             let formatter = DateFormatter()
-            formatter.locale = Locale(identifier: "en_US_POSIX")
-            formatter.timeZone = TimeZone(identifier: "GMT")
-            formatter.dateFormat = "MMM d"
+            formatter.setLocalizedDateFormatFromTemplate("MMM d")
```

Comment:
Can you add a comment above this, reminding me about what this format looks like?

// e.g., Jan 23